### PR TITLE
Small changes to improve readability

### PR DESCRIPTION
**Pull request by parfa30**
Many files changes with few significant changes

### CODE REVIEW

File: py/desiperf/instperfapp/pages/positioner.py [hunk 1]
```diff
@@ -29,15 +29,16 @@ def __init__(self, datahandler):
         posfiles = glob.glob(os.getcwd()+'/instperfapp/data/per_fiber/*.csv')
         self.pos_list = [os.path.splitext(os.path.split(posf)[1])[0] for posf in posfiles] #np.linspace(0,4999,5000)
         self.x_options = ['OFFSET_0','OFFSET_FINAL']
```
Comment (paulmartini) [inline py/desiperf/instperfapp/pages/positioner.py:32]: The .csv files do not have a 'OFFSET_FINAL' column

Comment (parfa30) [reply py/desiperf/instperfapp/pages/positioner.py:32]: Great catch! I believe they are OFFSET_2 or similar. I can make this quick change!

File: py/desiperf/instperfapp/pages/positioner.py [hunk 2]
```diff
@@ -29,15 +29,16 @@ def __init__(self, datahandler):
...
-        self.y_options = ['TARGET_RA', 'TARGET_DEC','FIBERASSIGN_X', 'FIBERASSIGN_Y','EXPOSURE','total_move_sequences',
+        self.y_options = ['datetime','EXPOSURE','TARGET_RA', 'TARGET_DEC','FIBERASSIGN_X', 'FIBERASSIGN_Y','total_move_sequences',
                         'airmass','mirror_temp','truss_temp','air_mirror_temp_diff','wind_speed','wind_direction',
```
Comment (paulmartini) [inline py/desiperf/instperfapp/pages/positioner.py]: airmass,  air_mirror_temp_diff, wind_speed, and several other options are not in the csv files.

File: py/desiperf/instperfapp/pages/positioner.py (hunk 1 above)
Comment (paulmartini) [reply py/desiperf/instperfapp/pages/positioner.py:32]: Yes, it is 'OFFSET_2'

**Review by paulmartini — APPROVED** [review]
A few comments on future items to fix:
  - in 'Positioner Accuracy' some of the Option 2 items are missing, and the plot does not update if the first positioner (595) is selected.
 - in 'Guiding' and 'Throughput' the plots do not update when changing menu items,  although the Data Overview text often does
 - in 'Detector Noise' I'm getting "NameError: name 'get_camera_attributes' is not defined" when I try Option 2 == CAMERA_TEMP. It isn't obvious to me why this is happening.

**Comment by paulmartini** [comment]
Hi @parfa30 - I propose I merge this, and then we track various issues to fix. What do you think?

**Comment by parfa30** [comment]
Hi Paul, that sounds good to me. I expect there are a ton more small changes that need to be made both to the data and to the app. If we just keep track of them all I can keep working through them and improving them

**Pull request merged**